### PR TITLE
Rename CSS class interface properties container and loading

### PR DESCRIPTION
**Pull request by alextaing**
This PR aims to make the naming convention for container and loading styling in CSS class interfaces more consistent by taking the new convention: `classNameContainer` and `classNameLoading`.

See this [slack post](https://yext.slack.com/archives/C032CKFARGS/p1655919900862129?thread_ts=1655919823.320149&cid=C032CKFARGS) and closed PR https://github.com/yext/answers-react-components/pull/192 for more information:

J=SLAP-2172
TEST=auto

Storybook shows that no visual changes were made to the stying.

### CODE REVIEW

File: src/components/VerticalResults.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { Pagination, PaginationCssClasses } from './Pagination';
  */
 export interface VerticalResultsCssClasses extends PaginationCssClasses {
   results?: string,
```
Comment (tmeyer2115) [inline src/components/VerticalResults.tsx:12]: Is this the container class or no?

File: src/components/NumericalFacets.tsx [hunk 2]
```diff
@@ -11,7 +11,7 @@ import { isNumberRangeFilter } from '../models/NumberRangeFilter';
  * @public
  */
 export interface NumericalFacetsCssClasses extends FilterGroupCssClasses, RangeInputCssClasses {
-  container?: string,
+  numericalFacetsContainer?: string,
   divider?: string,
   rangeInputContainer?: string,
```
[the remark this answers is not in the document]
Comment (alextaing) [reply src/components/NumericalFacets.tsx]: Removed, thanks!

File: src/components/NumericalFacets.tsx [hunk 3]
```diff
@@ -64,13 +64,13 @@ export function NumericalFacets({
     const { rangeInputContainer, rangeInputDivider, ...remainingCustomCssClasses } = customCssClasses;
     return {
       ...remainingCustomCssClasses,
-      container: rangeInputContainer,
+      rangeInputContainer,
       divider: rangeInputDivider
     };
```
[the remark this answers is not in the document]
Comment (alextaing) [reply src/components/NumericalFacets.tsx]: Got it.  Removed rangeInputContainer and kept divider for now.

File: src/components/ResultsCount.tsx [hunk 4]
```diff
@@ -14,8 +14,8 @@ import { useComposedCssClasses } from '../hooks/useComposedCssClasses';
  * @public
  */
 export interface ResultsCountCssClasses {
-  resultCountText?: string,
-  resultCountText___loading?: string
+  resultCountContainer?: string,
```
Comment (nmanu1) [inline src/components/ResultsCount.tsx:17]: nit: can these be renamed to `resultsCountContainer` and `resultsCount___loading` (add "s" after "result") to match the component name?

Comment (alextaing) [reply src/components/ResultsCount.tsx]: done!

**Comment by oshi97** [comment]
could you actually change the triple underscore to single underscore for everything
https://yext.slack.com/archives/C032CKFARGS/p1656356436920839

**Review by oshi97 — APPROVED** [review]
bet, don't forget to update the PR description. The desc still has the underscores

**Review by oshi97 — APPROVED** [review]

**Pull request merged**